projects/api/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import CreateUserDto from '@/user/dto/create-user.dto';
+
+import AuthService from './auth.service';
+import LoginDto from './dto/login.dto';
+import Auth from './entities/auth.entity';
+
+@ApiTags('Auth')
+@Controller('auth')
+class AuthController {
```

Comment:
Auth? Please fill the name at all, something like this: AuthenticationController

Reply:
hmm I was not able to see... strategies. those should be used for authentication...

Reply:
all the strategies are in a directory named strategy, and its Auth is only the name that appears in the swagger not more else